lib/gcloud/resource_manager/project.rb

```diff
@@ -359,6 +359,153 @@ def undelete
       end
 
       ##
+      # Gets the access control policy. Read
```

Comment:
What about adding a section to the overview documentation and linking to the [Cloud IAM docs](https://cloud.google.com/iam/) from there?